lib/suchthat.js

```diff
@@ -43,8 +44,8 @@ function suchthat(arb, userenv, predicate) {
       return arb.shrink(x).filter(predicate);
     }),
 
-    show: arb.show,
-  };
+    show: arb.show
```

Comment:
please leave the trailing comma (even `eslint` doesn't seem to complain about that)